qa/tasks/vstart_runner.py

```diff
@@ -1338,6 +1342,10 @@ def exec_test():
     if opt_create_cluster_only:
         return
 
+    global FSID
+    FSID = remote.run(args=[os.path.join(BIN_PREFIX, 'ceph'), 'fsid'],
```

Comment:
I guess this line should be written above line #1340 as we are calling LocalContext() there and FSID is not defined till then.

Reply:
No this place is fine.

Reply:
Okay, I see what you meant, there an instance of LocalContext being created at line 1340. Nice catch, thanks! I'll take care of it.